File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let sum = 0
    for (let i=0; i <= nums.length; i++) {
        sum += i
    }
    for (const n of nums) {
        sum -= n
    }
    return sum
};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        cur_sum = 0
        for i in range(0, len(nums) + 1):
            cur_sum += i
        for n in nums:
            cur_sum -= n
        return cur_sum
        
        #
        
        for i in range(0, len(nums)):
            if nums[i] != i:
                return i
        return len(nums)